src/ring_theory/power_series/basic.lean

```diff
@@ -780,10 +780,22 @@ by simp only [coeff_coe, mv_power_series.coeff_mul, coeff_mul]
   ((C a : mv_polynomial σ R) : mv_power_series σ R) = mv_power_series.C σ R a :=
 coe_monomial _ _
 
+@[simp, norm_cast] lemma coe_bit0 :
+  ((bit0 φ : mv_polynomial σ R) : mv_power_series σ R) = bit0 (φ : mv_power_series σ R) :=
+coe_add _ _
+
+@[simp, norm_cast] lemma coe_bit1 :
+  ((bit1 φ : mv_polynomial σ R) : mv_power_series σ R) = bit1 (φ : mv_power_series σ R) :=
+by rw [bit1, bit1, coe_add, coe_one, coe_bit0]
```

Comment:
Not fewer characters, but a simpler proof would be `coe_to_mv_power_series.ring_hom.map_bit1`

Reply:
That involves moving it after the ring hom instance though, it seems slightly more organised to leave these coe lemmas all in one block as much as possible to me

Reply:
Usually the approach is to prove `coe_one` and `coe_mul`, build the ring_hom, then prove everything else using it. See for instance how `coe_sum` is usually proved.

Reply:
I don't this matters too much in this case.